ext_conf_template.txt

```diff
@@ -0,0 +1,4 @@
+# cat=basic/login; type=boolean; label=LLL:EXT:cvc_webauthn/Resources/Private/Language/ExtConfTemplate.xlf:login.secondFactorLogin
+secondFactorLogin = 1
```

Comment:
If the default value is `1`, then it is a breaking change. To keep the functionality the same before and after this change, the default value should be `0`. Because not asking for the password during login is the current behaviour in the `master` branch.

Reply:
Makes sense. Changed it back to 0. Thanks